src/frontend/src/lib/components/ui/InputText.svelte

```diff
@@ -15,7 +15,4 @@
 	{placeholder}
 	spellcheck={false}
 	autocomplete="off"
-	on:nnsInput
->
-	<slot name="inner-end" slot="inner-end" />
-</Input>
+/>
```

Comment:
In case of `InputText` the `on:nnsInput` is currently unused therefore I removed it.